### ISSUE TITLE
Feature: Keyboard support

### ISSUE THREAD

**Issue by AlexLnz** [msarsha/ng2-right-click-menu#41]
Hi,

When opening, the first item in the menu should be selected and the menu should get keyboard focus.
With  \<CursorDown> and \<CursorUp> keys, the user can navigate through the menu. \<Enter> selects the current item. <Esc> closes the menu.
